src/components/ConfirmModal.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect } from 'react';
+import { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Modal, Button } from 'rsuite';
+
+interface ModalInput {
+  modalTitle: string;
+  modalBody: string;
+  callBackFunc: any;
+  isOpen: boolean;
+  closeModal: any;
+}
+
+export default function ConfirmModal({
+  modalTitle,
+  modalBody,
+  callBackFunc,
+  isOpen,
+  closeModal,
+}: ModalInput) {
+  const [open, setOpen] = useState<boolean>(false);
+  const history = useHistory();
+  useEffect(() => {
+    if (isOpen) {
+      setOpen(true);
+    } else {
+      setOpen(false);
+    }
+  });
```

Comment:
It's fine to use a passed `open/setOpen` from the parent. This set of lines can be removed.